### PR TITLE
Custom Code

### PR DESCRIPTION
**Pull request by ctf-umbraco**

### CODE REVIEW

File: Umbraco-Uno/Uno-pedia/Settings/Custom-Code/index.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+1. Navigate to your project in the portal
+
+2. Then go to settings
+
+3. Select ***Enable Custom Code***
+
+![this image shows what the menu looks like](images/Enable-Custom-Code.png)
+
+4. You will be taken to a new page with a disclaimer, telling you that once you enable custom code cannot turn it off again.
```
Comment (jonat123) [inline Umbraco-Uno/Uno-pedia/Settings/Custom-Code/index.md:18]:
```suggestion
4. You will be taken to a new page with a disclaimer, telling you that once you enable custom code you cannot turn it off again.
```

File: Umbraco-Uno/Uno-pedia/Settings/Custom-Code/index.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+versionFrom: 8.0.0
+---
+
+# Custom Code
+
+In Umbraco Uno, it is possible to enable Custom Code, if you are on a Standard/Professional plan.
```
Comment (jonat123) [inline Umbraco-Uno/Uno-pedia/Settings/Custom-Code/index.md:7]:
```suggestion
In Umbraco Uno, it is possible to enable Custom Code, if you are on a Standard or Professional plan.
```

File: Umbraco-Uno/Uno-pedia/Settings/Custom-Code/index.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+The way you activate this feature is very straight forward:
+
+1. Navigate to your project in the portal
+
+2. Then go to settings
```
Comment (jonat123) [inline Umbraco-Uno/Uno-pedia/Settings/Custom-Code/index.md:12]:
```suggestion
2. Go to settings
```

File: Umbraco-Uno/Uno-pedia/Settings/Custom-Code/index.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+![this image shows what the menu looks like](images/Enable-Custom-Code.png)
+
+4. You will be taken to a new page with a disclaimer, telling you that once you enable custom code you cannot turn it off again.
+
+5. If you are sure you want to enable custom code then go ahead and click the green button saying ***Enable Custom Code***
+
+![this image shows what the disclimer page looks like](images/warning-page.png)
+
+After Enabling Custom code, you will be able to enter the settings section and the packages section on your Development and Staging environments(not on live).
+Furthermore, you will gain access to Git for Development and Staging.
```
Comment (jonat123) [inline Umbraco-Uno/Uno-pedia/Settings/Custom-Code/index.md:25]: Maybe be more specific than they get access to git, maybe mention they can clone down the project.

File: Umbraco-Uno/Uno-pedia/Settings/Custom-Code/index.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
+---
+versionFrom: 8.0.0
+---
+
+# Custom Code
+
+In Umbraco Uno, it is possible to enable Custom Code, if you are on a Standard or Professional plan.
+The way you activate this feature is very straight forward:
```
Comment (Jay-umbr) [inline Umbraco-Uno/Uno-pedia/Settings/Custom-Code/index.md:8]: The way you activate this feature is very straight-forward:

File: Umbraco-Uno/Uno-pedia/Settings/Custom-Code/index.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+1. Navigate to your project in the portal
+
+2. Go to settings
+
+3. Select ***Enable Custom Code***
+
+![this image shows what the menu looks like](images/Enable-Custom-Code.png)
+
+4. You will be taken to a new page with a disclaimer, telling you that once you enable custom code you cannot turn it off again.
```
Comment (Jay-umbr) [inline Umbraco-Uno/Uno-pedia/Settings/Custom-Code/index.md:18]: 4. You will be taken to a new page with a disclaimer, telling you that once you enable custom code you cannot revert that change.

File: Umbraco-Uno/Uno-pedia/Settings/Custom-Code/index.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+5. If you are sure you want to enable custom code then go ahead and click the green button saying ***Enable Custom Code***
```
Comment (Jay-umbr) [inline Umbraco-Uno/Uno-pedia/Settings/Custom-Code/index.md:20]: 5. If you are sure that you want to enable custom code, go ahead and click the green button saying ***Enable Custom Code***

**Pull request merged**